docs/reference/storage-reference.md

```diff
@@ -1,5 +1,109 @@
+| `producer_names`   | **list[str]**<br/>Search for assets from any of the provided [producers](https://docs.up42.com/developers/api-glossary#producers).                                                                                                                                                                                                                                                                                                          |
+| `tags`             | **list[str]**<br/>Search for assets with any of the provided tags.                                                                                                                                                                                                                                                                                                                                                                          |
+| `sources`          | **list[str]**<br/>Search for assets from any of the provided sources. The allowed values:<br/><ul><li>`"ARCHIVE"`</li><li>`"TASKING"`</li><li>`"ANALYTICS"`</li><li>`"USER"`</li></ul>                                                                                                                                                                                                                                                      |
+| `search`           | **str**<br/>Search for assets that contain the provided search query in their name, title, or order ID.                                                                                                                                                                                                                                                                                                                                     |
+| `limit`            | **int**<br/>The number of assets on a result page.                                                                                                                                                                                                                                                                                                                                                                                          |
+| `sortby`           | **str**<br/>The property to sort by. The allowed values are:<br/><ul><li>`"id"`</li><li>`"workspaceId"`</li><li>`"accountId"`</li><li>`"createdAt"`</li><li>`"updatedAt"`</li><li>`"name"`</li><li>`"size"`</li><li>`"contentType"`</li><li>`"geospatialMetadataExtractionStatus"`</li><li>`"productId"`</li><li>`"orderId"`</li><li>`"producerName"`</li><li>`"collectionName"`</li><li>`"source"`</li>The default value is `"createdAt"`. |
```

Comment:
For sortby, descending, and return_json, please reuse the text from https://sdk.up42.com/reference/project-reference/#get_jobs

Reply:
Sortby values will be different here + please remove `"`

Reply:
`</li>The default value is "createdAt". |`
You need to close the list with `</ul>`